### PR TITLE
add elivepatch live ebuild

### PR DESCRIPTION
**Pull request by aliceinwire**
https://github.com/aliceinwire/elivepatch-server/issues/7

### CODE REVIEW

File: sys-apps/elivepatch-server/elivepatch-server-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+
+RDEPEND="
+	dev-python/werkzeug[${PYTHON_USEDEP}]
+	dev-python/flask[${PYTHON_USEDEP}]
+	dev-python/flask-restful[${PYTHON_USEDEP}]
+	=sys-kernel/kpatch-9999"
```
Comment (gktrk) [inline sys-apps/elivepatch-server/elivepatch-server-9999.ebuild]: Something like `>sys-kernel/kpatch-0.4.0` may be better here, which can only be provided by `-9999` at the moment anyway.

**Comment by aliceinwire** [comment]
removed x86 from livebuild

**Comment by gktrk** [comment]
Depends on https://github.com/gentoo/gentoo/pull/5666

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/4fdae0b0e6e...abec7c3e3c1. Thanks!

**Pull request closed without merging**